update get started with using kubernetes ingress (#1510)
vdaas/vald: docs/tutorial/get-started.md

## Prepare the Kubernetes Cluster

This tutorial requires the Kubernetes cluster.<br>

> Vald will run on Cloud Service such as GKE, AWS.

Reviewer: ```suggestion
Vald can run on public cloud services such as GKE, AWS.
```

Author: ```suggestion
Vald runs on public Cloud Services such as GKE, AWS.
```

> Vald runs on public Cloud Services such as GKE, AWS.

Author: ```suggestion
Vald runs on public Cloud Services such as GKE, EKS.
```

> Vald runs on public Cloud Services such as GKE, AWS.

Reviewer 2: LGTM:+1:

1. Clone the repository

   ```bash
   git clone https://github.com/vdaas/vald.git
   cd vald
   ```

1. Confirm which cluster to deploy

   ```bash
   kubectl cluster-info
   ```

> 1. Edit Values

Reviewer: ```suggestion
1. Edit Settings
```

Author: ```suggestion
1. Edit Configurations
```

>    You have to set the exact parameters for connecting Kubernetes ingress and vald-lb-gateway.

Reviewer: ```suggestion
   You have to set the extra parameters for connect to the vald-lb-gateway through ingress.
```

Author: ```suggestion
   Set the parameters for connecting to the vald-lb-gateway to Kubernetes ingress.
```

Reviewer: ```suggestion
   Set the parameters for connecting to the vald-lb-gateway through Kubernetes ingress.
```

Author: ```suggestion
   Set the parameters for connecting to the vald-lb-gateway through Kubernetes ingress to the external.
```
it' better and exact, isnt it?

Reviewer: I think it should be like 
```suggestion
   Set the parameters for connecting to the vald-lb-gateway through Kubernetes ingress from the external network.
```

   Please set the these parameters.

   ```bash
   vim example/helm/values.yaml
   ===
   ## vald-lb-gateway settings
   gateway:
     lb:
       ...
       ingress:
         enabled: true
         # TODO: Set your ingress host.
         host: localhost
         # TODO: Set annotations which you have to set for your k8s cluster.
         annotations:
           ...
   ```

   Note:<br>

>    If using port-forward instead of ingress, please set `gateway.lb.ingress.enabled` as `false`.

Reviewer: ```suggestion
   If you decided to use port-forwarding instead of ingress, please set `gateway.lb.ingress.enabled` to `false`.
```

>    If you decided to use port-forwarding instead of ingress, please set `gateway.lb.ingress.enabled` to `false`.

Author: ```suggestion
   If you decided to use port-forward instead of ingress, please set `gateway.lb.ingress.enabled` to `false`.
```
format vocabulary

   kubernetes           ClusterIP   10.43.0.1    <none>        443/TCP             9m29s
   vald-agent-ngt       ClusterIP   None         <none>        8081/TCP,3001/TCP   8m48s
   vald-discoverer      ClusterIP   None         <none>        8081/TCP,3001/TCP   8m48s
   vald-manager-index   ClusterIP   None         <none>        8081/TCP,3001/TCP   8m48s
   vald-lb-gateway      ClusterIP   None         <none>        8081/TCP,3001/TCP   8m48s
   ```

   </details>

## Run Example Code

In this chapter, you will execute insert vectors, search vectors, and delete vectors to your Vald cluster using the example code.<br>
The [fashion-mnist](https://github.com/zalandoresearch/fashion-mnist) is used as a dataset for indexing and search query.

The example code is implemented Go and using [vald-client-go](https://github.com/vdaas/vald-client-go), one of the official Vald client libraries, for requesting to Vald cluster.
Vald provides multiple language client libraries such as Go, Java, Node.js, Python, and so on.
If you are interested in, please refer to [SDKs](../user-guides/sdks.md).<br>

1.  Port Forward(option)

>     If you do not use Kubernetes Ingress, port-forward is required to make request from your local environment possible.

Reviewer: ```suggestion
    If you do not use Kubernetes Ingress, port-forward is required to make request from your local environment.
```

Author: ```suggestion
    If you do not use Kubernetes Ingress, port-forward is required to make requests from your local environment.
```
